test/ipfs-api/pubsub.spec.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-env mocha */
+/* eslint max-nested-callbacks: ['error', 8] */
+/* globals apiClients */
```

Comment:
Use just the factory to spawn multiple nodes (we want to avoid globals and setup scripts and migrate out of it entirely)